ons_alpha/bulletins/models.py

```diff
@@ -164,3 +171,14 @@ def previous_releases(self, request):
             context_overrides={"bulletins": previous},
             template="templates/pages/bulletins/previous_releases.html",
         )
+
+    @path("previous/v<int:version>/")
+    def previous_version(self, request, version):
```

Comment:
Issue: I believe it's the bulletin itself which has versions, as opposed to the series.